salt/utils/yamldumper.py

```diff
@@ -123,7 +106,4 @@ def safe_dump(data, stream=None, **kwargs):
     represented properly. Ensure that unicode strings are encoded unless
     explicitly told not to.
     """
-    if "allow_unicode" not in kwargs:
```

Comment:
same here

Reply:
since they are calling the dump from above, it is going to do the kwarg default above so this is extra code.  no use doing it twice when once will do.